ddtrace/ext/tags.go

```diff
@@ -33,6 +33,9 @@ const (
 	// HTTPURL sets the HTTP URL for a span.
 	HTTPURL = "http.url"
 
+	// HTTPUserAgent sets the HTTP user agent tag.
```

Comment:
```suggestion
	// HTTPUserAgent specifies the name of the tag which holds the HTTP user agent tag.
```
It doesn't _set it_ per se, it is not a function.

Reply:
Fixed in https://github.com/DataDog/dd-trace-go/pull/1286/commits/e989bc6a2b14146ed4a2186b599b1aed413c7699